extensions/codemagic/src/trigger-build.tsx

```diff
@@ -143,16 +144,16 @@ const TriggerBuildCommand = () => {
                         />
                       }
                     />
+                    <Action title="Refresh Apps and Branches" onAction={revalidate} icon={Icon.Repeat} />
                     {app.lastBuild && cancellableStatuses.includes(app.lastBuild.status) && (
                       <Action
                         title="Cancel Build"
                         onAction={async () => {
                           await cancelBuild(app.lastBuild!._id);
                         }}
-                        icon="x-mark-circle-16"
+                        icon={{ source: Icon.XMarkCircle, tintColor: Color.Red }}
```

Comment:
Here you can use ` style={Action.Style.Destructive}` intead of icon, that should make both an icon and style it 🙂